src/hooks/useMultiSelect.stories.mdx

```diff
@@ -41,22 +39,134 @@ export default function MultiSelectWithHookExample() {
   const multiSelectId = "multi-select-1";
   // Without this hook, you'd have to add handlers for onChange and onClear,
   // and also manage the selectedItems state yourself.
-  const { onChange, onClear, selectedItems } = useMultiSelect(
-    multiSelectId,
-    items
+  const { onChange, onClear, selectedItems } = useMultiSelect();
+
+  return (
+    <MultiSelect
+      id={multiSelectId}
+      variant="listbox"
+      items={items}
+      selectedItems={selectedItems}
+      onChange={(selectedItem) => onChange(selectedItem.id, multiSelectId)}
+      onClear={() => {
+        onClear(multiSelectId);
+      }}
+    />
   );
+}
+```
+
+## useMultiSelect Hook implementation in a `dialog` MultiSelect
+
+```jsx
+import React from "react";
+import MultiSelect from "./MultiSelect";
+import useMultiSelect from "./MultiSelect/useMultiSelect";
+
+export default function MultiSelectWithHookExample() {
+  const multiSelectId = "multi-select-1";
+  // Without this hook, you'd have to add handlers for onChange and onClear,
+  // and also manage the selectedItems state yourself.
+  const { onChange, onMixedStateChange, onClear, selectedItems } =
+    useMultiSelect();
 
   return (
     <MultiSelect
       id={multiSelectId}
       variant="listbox"
       items={items}
       selectedItems={selectedItems}
-      onChange={(selectedItem) => onChange(selectedItem.id)}
+      onChange={(selectedItem) => onChange(selectedItem.id, multiSelectId)}
+      onMixedStateChange={(selectedItem) =>
+        onMixedStateChange(selectedItem.id, multiSelectId, items)
+      }
       onClear={() => {
-        onClear();
+        onClear(multiSelectId);
       }}
     />
   );
 }
 ```
+
+## useMultiSelect Hook implementation in a `MultiSelectGroup`
+
+The data structure used to rendere two `MultiSelect` components in the following `MultiSelectGroup example:
+
+```jsx
+const multiSelectItems = [
+  {
+    id: "pets",
+    name: "Pets",
+    items: [
+      { id: "cat", name: "Cat" },
+      {
+        id: "dog",
+        name: "Dog",
+        children: [
+          { id: "corgy", name: "Corgy" },
+          { id: "german-sheperd", name: "German Sheperd" },
+          { id: "afghan-hound", name: "Afghan Hound" },
+        ],
+      },
+      { id: "rat", name: "Rat" },
+    ],
+  },
+  {
+    id: "tools",
+    name: "Tools",
+    items: [
+      { id: "hammer", name: "Hammer" },
+      {
+        id: "skrewdriver",
+        name: "Skrewdriver",
+        children: [
+          { id: "slottet", name: "Slotted Skrewdriver" },
+          { id: "phillips", name: "Phillips Skrewdriver" },
+          { id: "allen", name: "Allen Skrewdriver" },
+        ],
+      },
+      { id: "whisk", name: "Whisk" },
+    ],
+  },
+];
+```
+
+The implemntation of the `useMultiSelect` hook together with the `MultiSelectGroup` component.
+
+```jsx
+import React from "react";
+import MultiSelectGroup from "./MultiSelectGroup";
+import useMultiSelect from "./MultiSelect/useMultiSelect";
```

Comment:
Again, import from `@nypl/design-system-react-components`.

Reply:
fixed

Reply:
This update may not have been pushed.

Reply:
Completely missed the last one. It's fixed now.